runtime/clear_vmi/CMakeLists.txt

```diff
@@ -21,7 +21,6 @@
 ################################################################################
 
 add_library(j9vmi STATIC
-	j9vmi_harmony.c
```

Comment:
Seems j9vmi_harmony.c will need to be included when J9VM_OPT_HARMONY is enabled (as it must be for IBM Java 8 builds).

Reply:
Its not currently referenced from the module.xml (https://github.com/eclipse/openj9/blob/master/runtime/clear_vmi/module.xml#L37)  Also it would seem to duplicate symbols from j9vmi_j9.c, so I'm not actually sure how it was linking properly before

Reply:
Ok, it does appear to be an obsolete file, it was removed from the module.xml in 2011. Can you please delete j9vmi_harmony.c as well.

Reply:
FIxed